### PR TITLE
Fix retrieving ipa info containing embedded app bundle

### PR DESCRIPTION
**Pull request by ngs**
Moved implementation from https://github.com/fastlane/fastlane/pull/1046

This will ignore Info.plist of app bundle contained in the bundle, such as WatchOS application.

### CODE REVIEW

File: lib/fastlane_core/ipa_file_analyser.rb [hunk 1]
```diff
@@ -19,6 +20,8 @@ def self.fetch_app_version(path)
     def self.fetch_info_plist_file(path)
       Zip::File.open(path) do |zipfile|
         zipfile.each do |file|
+          next if File.basename(file.name).start_with?('._')
```
Comment (KrauseFx) [inline lib/fastlane_core/ipa_file_analyser.rb]: Could you add some documentation and examples on why that's necessary?

File: lib/fastlane_core/ipa_file_analyser.rb [hunk 2]
```diff
@@ -19,6 +20,8 @@ def self.fetch_app_version(path)
...
+          next if file.name.match(%r{(?:.+)\.app/.+\.app/Info.plist$})
```
Comment (KrauseFx) [inline lib/fastlane_core/ipa_file_analyser.rb]: Wouldn't it be saver to leave remove this? `(?:.+)\`

**Comment by ngs** [comment]
@KrauseFx Thanks for reviewing my code. Updated the code.

**Comment by KrauseFx** [comment]
Looks great, thanks @ngs. This will be in the next release :+1:

**Pull request merged**